examples/purchase-tester/src/main/AndroidManifest.xml

```diff
@@ -15,12 +15,19 @@
             android:theme="@style/AppTheme">
         <activity
                 android:name="com.revenuecat.purchasetester.MainActivity"
+                android:launchMode="singleTop"
```

Comment:
Had to do this to avoid the same activity from launching multiple times with the deep link... We might want to recommend developers to do the same.